tree/dataframe/src/RDataFrame.cxx

```diff
@@ -1916,6 +1916,211 @@ Non-finite numbers can be suppressed using Filter(), e.g.:
 df.Filter("std::isfinite(x)").Mean("x")
 \endcode
 
+### Translating TTree::Draw to RDataFrame 
+
+<table>
+<tr>
+   <td>
+      <b>TTree::Draw</b>
+   </td>
+   <td>
+      <b>ROOT::RDataFrame</b>
+   </td>
+</tr>
+<tr>
+   <td>
+~~~{.cpp}
+// Get the tree and Draw a histogram of x for selected y values
+auto *tree = file->Get<TTree>("myTree");
+tree->Draw("x", "y > 2");
+~~~
+   </td>
+   <td>
+~~~{.cpp}
+ROOT::RDataFrame df("myTree", file);
+auto h = df.Filter("y > 2").Histo1D("x");
+h->Draw();
+~~~
+   </td>
+</tr>
+<tr>
+   <td>
+~~~{.cpp}
+// Draw a histogram of "jet_eta" with the desired weight
+tree->Draw("jet_eta", "weight*(event == 1)");
+~~~
+   </td>
+   <td>
+~~~{.cpp}
+df.Filter("event == 1").Histo1D("jet_eta", "weight");
+
+// or the fully compiled version:
+df.Filter([] (ULong64_t e) { return e == 1; }, {"event"}).Histo1D<RVec<float>>("jet_eta", "weight");
+~~~
+   </td>
+</tr>
+<tr>
+   <td>
+~~~{cpp}
+// Draw a histogram using the method of the class that is being used. 
+tree->Draw("event.GetNtrack()");
+// note: this will not work in RDataFrame version
+// tree->Draw("GetNtrack()");
+
+~~~
+   </td>
+   <td>
+~~~{cpp}
+auto df1 = df.Define("NTrack","event.GetNtrack()");
+auto df2 = df1.Histo1D<int>("NTrack");
```

Comment:
If we're going for the shortest amount of characters possible:
```suggestion
auto df2 = df1.Histo1D("NTrack");
```

And we also need to Draw
```suggestion
df1.Histo1D("NTrack")->Draw();
```